smart.lock

```diff
@@ -1,31 +0,0 @@
-{
-  "meteor": {},
-  "dependencies": {
-    "basePackages": {
-      "router": {},
-      "chosen-package": {}
-    },
-    "packages": {
-      "router": {
-        "git": "https://github.com/tmeasday/meteor-router.git",
-        "tag": "v0.6.1",
-        "commit": "8ec75fec7affdefc787d19f23b36fd6d1d44ef04"
-      },
-      "chosen-package": {
-        "git": "https://github.com/Ore4444/chosen-package.git",
-        "tag": "v0.11.1",
-        "commit": "8d1a748bafaae8b6005a979ab7261ac14ff38052"
-      },
-      "page-js-ie-support": {
-        "git": "https://github.com/tmeasday/meteor-page-js-ie-support.git",
-        "tag": "v1.3.5",
-        "commit": "b99ed8380aefd10b2afc8f18d9eed4dd0d8ea9cb"
-      },
-      "HTML5-History-API": {
-        "git": "https://github.com/tmeasday/meteor-HTML5-History-API.git",
-        "tag": "v4.1.2",
-        "commit": "b5fca79f9ae8936e5f748a04f475295f4fa1cc7a"
-      }
-    }
-  }
-}
```

Comment:
Were you able to switch to using the `meteor` tool instead of `mrt` entirely so that you're getting rid of this?